crates/extensions/tedge-p11-server/src/proxy/server.rs

```diff
@@ -128,6 +129,34 @@ impl TedgeP11Server {
             // ensures the PKCS11 library is loaded and ready to serve signing requests. In
             // practice, this only occurs with a client calls TedgeP11Client::with_ready_check.
             Frame1::Ping => Frame1::Pong,
+
+            Frame1::CreateKeyRequest(request) => {
```

Comment:
This must explicitly be a new version.


```suggestion
            Frame2::CreateKeyRequest(request) => {
```